External/FEXCore/Source/Utils/ELFLoader.cpp

```diff
@@ -435,7 +435,7 @@ void ELFContainer::CalculateSymbols() {
       uint64_t offset = DynSymTabHeader->sh_offset + i * DynSymTabHeader->sh_entsize;
       Elf32_Sym const *Symbol =
           reinterpret_cast<Elf32_Sym const *>(&RawFile.at(offset));
-      if (ELF32_ST_VISIBILITY(Symbol->st_other) != STV_HIDDEN &&
+      if (ELF64_ST_VISIBILITY(Symbol->st_other) & 0x3 != STV_HIDDEN &&
```

Comment:
Name flipped from other one and oops on still doing & 3?

Reply:
oops